### ISSUE TITLE
License file

### ISSUE THREAD

**Issue by KFishner** [manucorporat/stats#2]
Hey there, we use your software in [Consul](github.com/hashicorp/consul) and need a license file to be able to officially use it. Any chance you could add one - MPL2 or Apache?

MPL2 example: https://github.com/hashicorp/consul/blob/master/LICENSE
Apache example: https://github.com/moby/moby/blob/master/LICENSE

**Comment by caniszczyk** [comment]
Yes, can you please add the LICENSE ? Apache v2.0 would be preferable.

**Comment by gnuphie** [comment]
Yes, an Apache v2.0 license would be great :)

**Comment by lopter** [comment]
Please see #3.
